src/lbaf/IO/lbsConfigurationValidator.py

```diff
@@ -159,7 +159,11 @@ def __init__(self, config_to_validate: dict, logger: Logger):
                          Optional("cluster_swap_rtol"): And(
                             float,
                             lambda x: x > 0.0,
-                            error="Should be of type 'float' and magnitude > 0.0"),
+                            error="Should be of type 'float' and > 0.0"),
+                         Optional("max_subclusters"): And(
+                            int,
+                            lambda x: x > 0.0,
+                            error="Should be of type 'int' and > 0"),
```

Comment:
Should we add a unit test for this new parameter?

Reply:
Yes please, that would be good

Reply:
Ok, I added the tests